src/pages/index.js

```diff
@@ -327,7 +326,8 @@ const Index = ({ data: fallbackData }) => {
   const userGlobalId = get(user, 'id')
   const globalTabCount = get(user, 'tabs')
   const [tabId] = useState(uuid())
-
+  const ThemeWrapper = useCustomTheme(user.causeId)
+  const classes = useStyles()
```

Comment:
@kmjennison i don't think we can use a theme hook bc the theme object in the useStyles() in the index page uses the theme object *above* this file in app.js.  Everything nested below this will get the new theme.  The HOC is still messy code so i'm going to refactor it but I think we're stuck with HOC

Reply:
or i guess we could pass in props to useStyles(props) and have the props be the new theme object

Reply:
still think i'm going to go with an HOC for now.  I'm getting weird rerenders in the hook